server.py

```diff
@@ -57,7 +57,7 @@
     async with ws_handler(request, ws_receivers) as ws:
         # This message is sent only this (just connected) client
         await ws.send_json(state.as_dict(map_name), dumps=json.dumps)
-        # Process messages from this client
+        # For cycle keeps the connection with client alive
```

Comment:
nesmažeš to "alive"? bez toho mi to přijde hezčí, to alive mi přijde takové divné :-D

Reply:
Nerada ;). Fráze "keep alive" je docela běžná v IT žargonu, když se mluví o vlastnostech spojení. My sice využíváme websockety, nikoli http, ale přijde mi to zde hezky popsané: https://en.wikipedia.org/wiki/HTTP_persistent_connection

Reply:
Jako je to možné, jelikož moje angličtina je taková všelijaká, prostě mi to zní to alive jakoby spíš k živým tvorům :-D